src/lcf/data.cpp

```diff
@@ -0,0 +1,64 @@
+/*
+ * This file is part of liblcf. Copyright (c) 2020 liblcf authors.
```

Comment:
Is OK to convert headers to Player? Code looks trivial enough.

Reply:
This header will go away very soon, so I'm inclined to leave it as is until then. But up to you guys. I don't care either way.